clang/test/SemaSYCL/unsupported_math.cpp

```diff
@@ -0,0 +1,26 @@
+// RUN: %clangxx --sycl %s -c -o a_kernel.bc -Xclang -verify
```

Comment:
I suggest using following command line:

// RUN: %clang_cc1 -fsycl-is-device -fsyntax-only -verify %s

Reply:
I used the command line suggested here. Thanks